Application/SchedulingLib/models/DatabaseModels.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SchedulingLib.models
+{
+    public class Building
+    {
+        public int BuildingId { get; set; }
+        public string BuildingName { get; set; }
+
+        public List<Rooms> Room { get; set; }
```

Comment:
These should be plural because they're lists
```suggestion
        public List<Rooms> Rooms { get; set; }
```